src/libaktualizr/utilities/utils.cc

```diff
@@ -52,134 +53,134 @@ const char *adverbs[] = {
-                       "Kluntjes",
-                       "Knaeckebrot",
-                       "Kniekuechle",
-                       "Knoedel",
-                       "Kohlroulade",
-                       "Krautfleckerl",
-                       "Kuerbiskernbrot",
-                       "Kuerbissuppe",
-                       "Lebkuchen",
-                       "Linsen",
-                       "Loeffelerbsen",
-                       "Magenbrot",
-                       "Marillenknoedel",
-                       "Maroni",
-                       "Marsch",
-                       "Marzipan",
-                       "Maultaschen",
-                       "Milliramstrudel",
-                       "Mischbrot",
-                       "Mohnnudeln",
-                       "Mohnpielen",
-                       "Mohnzelten",
-                       "Muesli",
-                       "Nussecke",
-                       "Nusstorte",
-                       "Palatschinke",
-                       "Pellkartoffeln",
-                       "Pfannkuchen",
-                       "Pfefferkuchen",
-                       "Pillekuchen",
-                       "Pommes",
-                       "Poschweck",
-                       "Powidltascherl",
-                       "Printen",
-                       "Prinzregententorte",
-                       "Pumpernickel",
-                       "Punschkrapfen",
-                       "Quarkkeulchen",
-                       "Quetschkartoffeln",
-                       "Raclette",
-                       "Radi",
-                       "Reibekuchen",
-                       "Reinling",
-                       "Riebel",
-                       "Roeggelchen",
-                       "Roesti",
-                       "Sauerkraut",
-                       "Schmalzkuchen",
-                       "Schmorgurken",
-                       "Schnippelbohnen",
-                       "Schoeberl",
-                       "Schrippe",
-                       "Schupfnudel",
-                       "Schuxen",
-                       "Schwammerlsuppe",
-                       "Schweineohren",
-                       "Sonnenblumenkernbrot",
-                       "Spaetzle",
-                       "Spaghettieis",
-                       "Spargel",
-                       "Spekulatius",
-                       "Springerle",
-                       "Spritzkuchen",
-                       "Stampfkartoffeln",
-                       "Sterz",
-                       "Stollen",
-                       "Streuselkuchen",
-                       "Tilsit",
-                       "Toastbrot",
-                       "Topfenstrudel",
-                       "Vollkornbrot",
-                       "Wibele",
-                       "Wickelkloesse",
-                       "Zimtwaffeln",
-                       "Zwetschkenroester",
-                       "Zwiebelkuchen"};
+static const std::array<const char *, 128> names = {"Allerlei",
```

Comment:
Really annoying that you have to specify the length. Is this another reasonable candidate for a NOLINT?

Reply:
Yes kinda annoying but I don't think we'll change these very often so it shouldn't be a big deal. And just calling `.size()` rather than the sizeof dance is already beneficial.